goserver/server/server.go

```diff
@@ -37,9 +48,18 @@ func (s *Server) handleLaunch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "missing iss in URL query parameters", http.StatusUnauthorized)
 		return
 	}
-	if iss[0] != s.AuthorizedFHIRURL {
+	if iss[0] != s.authorizedFHIRURL {
 		http.Error(w, fmt.Sprintf("unauthorized iss %s", iss[0]), http.StatusUnauthorized)
 		return
 	}
+
+	sess, err := s.sm.New(w, r)
+	if err != nil {
+		http.Error(w, "cannot create session", http.StatusInternalServerError)
+		return
+	}
+	sess.FHIRURL = iss[0]
```

Comment:
why not pass iss[0] as a parameter of New function? It would be less likely to be forgotten by accident?

Reply:
Done.

Reply:
not done yet or forgot to upstream local commit?